core/templates/dev/head/pages/splash/splash.html

```diff
@@ -121,25 +121,26 @@ <h1 class="oppia-splash-h1" translate="I18N_SPLASH_TITLE"></h1>
         </div>
 
         <div class="oppia-splash-button-container">
+          <button type="button" ng-click="onClickBrowseLibraryButton()"
+                  class="btn oppia-splash-button oppia-splash-button-browse oppia-transition-200"
+                  style="bottom: 210px; z-index: 10;"
+                  translate="I18N_ACTION_BROWSE_LESSONS">
+          </button>
+
           <!-- User must complete the registration process. -->
```

Comment:
Do we need this? 
If so are we expecting that above button below this comment (as earlier)?

Reply:
Oh, I see, now I understand. Thanks!